### DISCUSSION TITLE
Unauthorized production of video and audio

### DISCUSSION THREAD

**Discussion by Kullorn119** [community/community#84708] [Other Feature Feedback, Questions, & Ideas]
I can't seem to understand any of github I need this video and audio recordings to stop .I have never signed any paper work or never been personally informed of anythis

**Comment by samus-aran** [comment]
Hi @Kullorn119 ! We hope you found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
